### ISSUE TITLE
Lab4 output

### ISSUE THREAD

**Issue by Faur** [DeepLearningDTU/02456-deep-learning-with-PyTorch#6]
> The output of the net is softmaxed so it’s probabilities, but then we use a loss function that assumes them to be scores and applies softmax again. This is not a big problem in practice, but in principle it doesn’t make sense, and it clashes with the score given by Kaggle if you upload the results, because as far as I understand Kaggle assumes the outputs to be scores, whereas we have probabilities.

**Comment by Faur** [comment]
Fixed in [cf461d004cbc40f839640a5e8207346e2e191b2d](https://github.com/DeepLearningDTU/02456-deep-learning-with-PyTorch/commit/cf461d004cbc40f839640a5e8207346e2e191b2d)
